app/controllers/detail-viewers.ts

```diff
@@ -835,7 +840,9 @@ function getFtpaAppellantDecisionDetails(req: Request, res: Response, next: Next
     const ftpaApplicationDate = req.session.appeal.ftpaAppellantApplicationDate;
     const ftpaDecision = req.session.appeal.ftpaAppellantDecisionOutcomeType || req.session.appeal.ftpaAppellantRjDecisionOutcomeType;
     const ftpaDecisionAndReasonsDocument = req.session.appeal.ftpaAppellantDecisionDocument;
+    const ftpaR35AppellantDocument = [req.session.appeal.ftpaR35AppellantDocument];
     const ftpaDecisionDate = req.session.appeal.ftpaAppellantDecisionDate;
+    const ftpaSetAsideFeatureEnabled: boolean = await LaunchDarklyService.getInstance().getVariation(req, FEATURE_FLAGS.DLRM_SETASIDE_FEATURE_FLAG, false);
```

Comment:
I think we use flag with AiP(we created another one with Fee)

Reply:
No, there are different feature flags per feature, this code belongs to set-aside feature so he is using the correct one